test/unit/utils.html

```diff
@@ -36,6 +46,50 @@
     delete window.el2;
   });
 
+  suite('DOM querying', function() {
+
+    test('$$', function () {
+      var byTag = window.el1.$$('span');
+      assert(byTag, 'querying local dom by tag selector returns first element of given tag-name');
+      assert(!window.el1.$$('div'), 'querying local dom for non-existent tag returns no element');
+
+      var byClass = window.el1.$$('.class-selectable');
+      assert(byClass, 'querying local dom by class selector returns first element with given class');
+      assert(!window.el1.$$('.class-nosuchthing'), 'querying local dom by class selector returns no element if no elements have given class');
+
+      var byId = window.el1.$$('#id-selectable');
+      assert(byId, 'querying local dom by id selector returns element with given id');
+      assert(!window.el1.$$('#id-nosuchthing'), 'querying local dom by id selector returns no element if given id does not exist');
+    });
+
+    test('$$$', function () {
+      var byTag = window.el1.$$$('span');
+      assert(byTag.length === 3
+              && byTag[0].tagName === 'SPAN'
```

Comment:
It is better to split these asserts into seperate asserts. When one of the conditions is false, the error message is more intuitive and easier to debug.